### PR TITLE
Trim newline at the end of the file

### PR DESCRIPTION
**Pull request by mykola-kobets-epam**

### CODE REVIEW

File: include/aos/common/tools/string.hpp [hunk 1]
```diff
@@ -163,6 +163,22 @@ class String : public Array<char> {
         return Resize(from - begin());
     }
 
+    /**
+     * Removes trailing characters.
+     *
+     * @param chars characters to be removed.
+     */
+    void RTrim(const String& chars) {
```
Comment (al1img) [inline include/aos/common/tools/string.hpp:171]: Rename to `RightTrim`, add `LeftTrim` and `Trim` (both).

File: include/aos/common/tools/fs.hpp [hunk 2]
```diff
@@ -421,7 +421,14 @@ class FS {
             return err;
         }
 
-        return text.Resize(buff.Size());
+        err = text.Resize(buff.Size());
```
Comment (al1img) [inline include/aos/common/tools/fs.hpp:424]: Trim should be done  in caller. ReadFileToString should read what ever is in the file.

File: include/aos/common/tools/string.hpp (hunk 1 above)
Comment (mykola-kobets-epam) [reply include/aos/common/tools/string.hpp]: fixed

File: include/aos/common/tools/fs.hpp (hunk 2 above)
Comment (mykola-kobets-epam) [reply include/aos/common/tools/fs.hpp]: removed PR

**Review by al1img — APPROVED** [review]

**Pull request merged**